tests/e2e/exec-env-vars-repro.test.ts

```diff
@@ -0,0 +1,195 @@
+import {
```

Comment:
Is this test intended to be checked-in? Seems to be a mix of testing pre-change behaviour vs new